module/programs/helper.js

```diff
@@ -509,29 +520,84 @@ module.exports = class ProgramsHelper {
 					}
 				}
 
+				let updateObject = { $addToSet: {} }
+				let validationExcludedEntitiesKeys = []
+				if (organizations && userDetails.userInformation.roles.includes(CONSTANTS.common.ADMIN_ROLE)) {
```

Comment:
@MallanagoudaB even if we don't have tenantadmin allow tenant admin role also for this flow .in future it will come handy